src/freva/cli/history.py

```diff
@@ -58,20 +58,26 @@ def __init__(
             "--since",
             default=None,
             type=str,
-            help="Retrieve entries older than date",
+            help="Retrieve entries older than date.",
         )
         self.parser.add_argument(
             "--until",
             default=None,
             type=str,
-            help="Retrieve entries newer than date",
+            help="Retrieve entries newer than date.",
         )
         self.parser.add_argument(
             "--entry-ids",
             default=None,
             type=str,
             nargs="+",
-            help="Select entry id(s) (e.g. --entry-ids 1 --entry-ids 2 )",
+            help="Select entry id(s) (e.g. --entry-ids 1 --entry-ids 2 ).",
+        )
+        self.parser.add_argument(
+            "--all-users",
```

Comment:
As my comment above. I think it would be better to make this a user flag.

Reply:
I will open a new branch for this, I reverted the changes and renamed this branch to user-all, closed making sure that at least this flag works well thru tests and docs